Cargo.toml

```diff
@@ -345,7 +345,7 @@ reth-primitives-traits = { path = "crates/primitives-traits", default-features =
 reth-provider = { path = "crates/storage/provider" }
 reth-prune = { path = "crates/prune/prune" }
 reth-prune-types = { path = "crates/prune/types" }
-reth-revm = { path = "crates/revm", default-features = false }
```

Comment:
Why in this case are we removing default-features = false?

Reply:
this is a workspace wide issue and handling features needs special care, I'm still considering options here on what's the solution. and I want to change this in a special pr

Reply:
thank you for clarifying